src/main/java/io/vertx/core/http/impl/Http1xClientConnection.java

```diff
@@ -59,7 +59,7 @@
  *
  * @author <a href="http://tfox.org">Tim Fox</a>
  */
-class Http1xClientConnection extends Http1xConnectionBase implements HttpClientConnection {
+public class Http1xClientConnection extends Http1xConnectionBase implements HttpClientConnection {
```

Comment:
does still this needs to be changed to public visibility ?

Reply:
Nope, forgot to remove that